repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -407,27 +407,43 @@ def filter_out_out_pkgs(event_in_pkgs, event_out_pkgs):
     return {k: v for k, v in event_in_pkgs.items() if k not in event_out_pkgs}
 
 
-def filter_out_pkgs_in_blacklisted_repos(to_install):
+SKIPPED_PKGS_MSG = (
+    'packages will be skipped because they are available only in '
+    'RHEL 8 repositories that are intentionally excluded from the '
+    'list of repositories used during the upgrade. '
+    'See the report message titled "Excluded RHEL 8 repositories" '
+    'for details.\n The list of these packages (repositories: {}):'
+)
+
+
+def filter_out_pkgs_in_excluded_repos(to_install):
     """
-    Do not install packages that are available in blacklisted repositories
+    Do not install packages that are available in excluded repositories
 
-    No need to filter out the to_keep packages as the blacklisted repos won't get enabled - that is ensured in the
+    No need to filter out the to_keep packages as the excluded repos won't get enabled - that is ensured in the
     setuptargetrepos actor. So even if they fall into the 'yum upgrade' bucket, they won't be available thus upgraded.
     """
     # FIXME The to_install contains just a limited subset of packages - those that are *not* currently installed and
-    # are to be installed. But we should also warn about the packages that *are* installed.
-    blacklisted_pkgs = set()
-    blacklisted_repos = get_repositories_blacklisted()
+    #   are to be installed. But we should also warn about the packages that *are* installed.
+    excluded_pkgs = set()
+    excluded_repos = get_repositories_excluded()
     for pkg, repo in to_install.items():
-        if repo in blacklisted_repos:
-            blacklisted_pkgs.add(pkg)
+        if repo in excluded_repos:
+            excluded_pkgs.add(pkg)
 
-    for pkg in blacklisted_pkgs:
+    for pkg in excluded_pkgs:
         del to_install[pkg]
 
-    if blacklisted_pkgs:
-        report_skipped_packages('packages will not be installed due to blacklisted repositories:',
-                                blacklisted_pkgs)
+    if excluded_pkgs:
+        report_skipped_packages(
+            message=SKIPPED_PKGS_MSG.format(', '.join(excluded_repos)),
+            packages=excluded_pkgs,
+            remediation=(
+                'AT YOUR OWN RISK!!! Use leapp upgrade {}'.format(' '.join(
+                    ['--enablerepo ' + repo for repo in excluded_repos])
```

Comment:
Have you verified that this really works as expected? I.e. that the list of excluded repositories is not used in any actor?

Reply:
@bocekm thanks for pointing out. I took it from somewhere (already not remember) as a fact. After testing this option, it reveals as not working at all. So the fix implemented now